mlflow/sklearn.py

```diff
@@ -8,10 +8,14 @@
 :py:mod:`mlflow.pyfunc`
```

Comment:
maybe we should create a directory named `sklearn` and seprate code into multiple files otherwise `mlflow/sklearn.py` would become very large and harder to maintain.

Reply:
+1. Let's do this!